packages/sc-deployer/src/index.ts

```diff
@@ -29,6 +33,51 @@ interface IDeploymentInfo {
   events?: IEvent[];
 }
 
+/**
+ * Check the balance
+ *
+ * @param web3Client - an initialized web3 client
+ * @param account - the wallet whose balance is being checked
+ * @param requiredBalance - the required balance to check against 
+ * @throws if the given account has insufficient founds
+ */
+async function checkBalance(web3Client: Client, account: IAccount, requiredBalance: number) {
+  if (account.address === null) {
+    throw new Error('Account has no address.');
+  }
+  const balance = await web3Client.wallet().getAccountBalance(account.address as string);
+  console.log(`Wallet Address: ${account.address} has balance (candidate, final) = (${balance?.candidate.rawValue()}, ${balance?.final.rawValue()}) MAS`);
+  if (!balance?.final || !parseFloat(balance.final.rawValue().toString()) || balance.final.rawValue().lt(requiredBalance)) {
```

Comment:
i dont think this parseFloat condition is usefull here

Reply:
absolutely, good catch. I have removed it